library/kani/src/lib.rs

```diff
@@ -64,6 +64,17 @@ pub const fn assert(_cond: bool, _msg: &'static str) {
     }
 }
 
+/// Creates a cover property with the specified condition and message.
```

Comment:
Can you please expand a bit on what a cover property is?

Reply:
Done